ignite/services/network/networkchain/init.go

```diff
@@ -105,7 +105,7 @@ func (c *Chain) initGenesis(ctx context.Context) error {
 			return err
 		}
 	case c.genesisConfig != "":
-		c.ev.Send("Fetching custom Genesis from Config", events.ProgressUpdate())
+		c.ev.Send("Fetching custom Genesis from ChainConfig", events.ProgressUpdate())
```

Comment:
Maybe this would make more sense to the users:
```suggestion
		c.ev.Send("Fetching custom genesis from chain config", events.ProgressUpdate())
```